test/e2e/support/utils/clear-local-storage.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * Clear localStorage
```

Comment:
Can we use the form of 3rd person for consistency and end the sentence with a full stop?

`Clears the local storage.`

Reply:
Updated, thank you.